main.py

```diff
@@ -30,10 +29,6 @@ def load() -> None:
         bot.load_extension(f"cogs.{cog}")
 
 
-def main() -> None:
+if __name__ == "__main__":
```

Comment:
On the topic of the main file, `if __name=="__main__"` is not needed because the script is only ran directly and is not meant to be imported.